src/caches/ssprk_caches.jl

```diff
@@ -34,7 +51,28 @@ alg_cache(alg::SSPRK22,u,rate_prototype,uEltypeNoUnits,uBottomEltypeNoUnits,tTyp
   step_limiter!::StepLimiter
 end
 
-struct SSPRK33ConstantCache <: OrdinaryDiffEqConstantCache end
```

Comment:
you can probably make this cache and the previous the same?

Reply:
we need to be careful about the ones that have a different interpolation though.

Reply:
Yeah we can have same caches but different constructors (I mean using inheritence).